### PR TITLE
Docker build script

### PR DESCRIPTION
**Pull request by MyDigitalLife**
I created a script that performs a make with docker. My work machine is a Windows 10 machine most of the time so make files are not supported by default. That's why I created a small docker image and a .sh script to build and run a docker image that does the build process for me.

Simple and easy. Just run build.sh in the docker folder if you have docker installed. I thought it would be nice to include this in the project. I tested it on windows 10 and on Ubuntu and both work like a charm.

### CODE REVIEW

**Comment by k4k** [comment]
@kris-nova Would you be opposed to standardizing the build on docker? It would provide some added benefits around build environment sanitization and standards.

File: docker/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo Building container
+docker build -t gobuilder "$(pwd)"
+echo Running make script
+docker run --rm -v "/$(pwd)/.."://usr/src/myapp -w //usr/src/myapp gobuilder make
```
Comment (kris-nova) [inline docker/build.sh]: Can we please mount to `/go/src/github.com/kris-nova/kubicorn` and have `$GOPATH=/go`?

**Comment by kris-nova** [comment]
@k4k I am all for having a docker build,

although I wouldn't want it to replace local build logic, but rather live next to it. Does that make sense at all? If you think it's a good idea please PR it and/or open an issue and we can go from there!

**Comment by MyDigitalLife** [comment]
I agree with @kris-nova that the docker build should not replace the existing process. Not everybody want to install docker or know what it is/does. 

I've added a remove image option and a verbose option. Its now almost completely silent by default and outputs everything when adding -v or --verbose. I can also add the documentation from #78 and add some extra text about the docker build.

File: docker/build.sh (hunk 1 above)
Comment (MyDigitalLife) [inline docker/build.sh]: This has been updated. And the GOPATH variable is /go by default (https://github.com/docker-library/golang/blob/2a15dfff04accfd31c2a45b3bb7423aa86aa2d60/1.8/jessie/Dockerfile line 45).

File: docker/build.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+VERBOSE=FALSE
+VERBOSE_DOCKER_RUN=""
+VERBOSE_DOCKER_BUILD="-q"
+REMOVE_IMAGE=FALSE
+
+docker >/dev/null 2>&1 || { echo >&2 "Docker is required but it's not installed. Aborting."; exit 1; }
```
Comment (k4k) [inline docker/build.sh]: Minor nit-pick but I don't like that we're blindly running whatever `docker` is in $PATH here just to check that it's available. Can we just `which docker > /dev/null || {...}` here instead?

**Comment by kris-nova** [comment]
LGTM - thanks everyone for your help

**Pull request merged**